Formula/go@1.11.rb

```diff
@@ -22,18 +22,28 @@ class GoAT111 < Formula
 
   # Don't update this unless this version cannot bootstrap the new version.
   resource "gobootstrap" do
-    url "https://storage.googleapis.com/golang/go1.7.darwin-amd64.tar.gz"
+    if OS.mac?
+      url "https://storage.googleapis.com/golang/go1.7.darwin-amd64.tar.gz"
+      sha256 "51d905e0b43b3d0ed41aaf23e19001ab4bc3f96c3ca134b48f7892485fc52961"
+    elsif OS.linux?
+      url "https://storage.googleapis.com/golang/go1.7.linux-amd64.tar.gz"
+      sha256 "702ad90f705365227e902b42d91dd1a40e48ca7f67a2f4b2fd052aaa4295cd95"
+    end
     version "1.7"
-    sha256 "51d905e0b43b3d0ed41aaf23e19001ab4bc3f96c3ca134b48f7892485fc52961"
   end
 
   def install
+    ENV["CGO_ENABLED"] = "1"
```

Comment:
@gcla Should this change be on both OSes? What does it do?

Reply:
hi @zbeekman - on Linux, without this change, I ran into the following error:

```
Last 15 lines from /home/gcla/.cache/Homebrew/Logs/go@1.11/02.go:      
2019-05-22 18:25:48 -0400                                                                                                                        
                                                                                                                                                         
/home/gcla/.linuxbrew/Cellar/go@1.11/1.11.6/bin/go                                    
install                                                         
-race                                     
std                                                                        
                                                                        
go install: -race requires cgo; enable cgo by setting CGO_ENABLED=1
```

But it seems it's not required on macOS - I had a friend with a mac try to install go@1.11 today and it seemed to work fine. I also should've split this PR into two, with separate logs for each. Apologies!